src/python_testing/TC_XX_1_1.py

```diff
@@ -0,0 +1,138 @@
+#
+#    Copyright (c) 2022 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import logging
+
+from matter_testing_support import MatterBaseTest, async_test_body, default_matter_test_main
+from mobly import asserts
+
+standard_range_prefix = 0x0000
+manufacturer_code_min_prefix = 0x0001
+manufacturer_code_max_prefix = 0xFFF0
+test_vendor_min_prefix = 0xFFF1
+test_vendor_max_prefix = 0xFFF4
+invalid_vendor_min_prefix = 0xFFF5
+invalid_vendor_max_prefix = 0xFFFF
+
+attribute_id_range_min = 0x0000
+attribute_id_range_max = 0x4FFF
+attribute_global_range_min = 0xF000
+attribute_global_range_max = 0xFFFE
+attribute_id_invalid_max = 0xFFFF
+
+# event_id_range_min = 0x00
```

Comment:
remove commented out code.
For constants I would prefer `UPPER_SNAKE_CASE` throughout (it seems IDs are like that but min/max are not.

Instead of having separate constants for min/max, how about merging them into one thing. Supposedly range allows for fixed sizeso you can do:

```py
MANUFACTURER_CODE_RANGE = range(0x0001, 0xFFF0 + 1)
TEST_VENDOR_RANGE = range(0xFFF1, 0xFFF4 + 1)
# ...
```

and then check range membership

Reply:
I have no problem with this change, I mainly tried to match the way https://github.com/project-chip/connectedhomeip/pull/28902/files is written.

Reply:
I have tried reworking the test script around your suggestion, let me know if this is acceptable.

Maybe we should consider doing a similar approach in IDM-10.1 once we merge these two test scripts.